[2023-04-06T18:44:28.778+0400] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 10X_sequence.archive_run_dir testrun_06_04_2023 18_44_01 [queued]>
[2023-04-06T18:44:28.790+0400] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 10X_sequence.archive_run_dir testrun_06_04_2023 18_44_01 [queued]>
[2023-04-06T18:44:28.790+0400] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-06T18:44:28.790+0400] {taskinstance.py:1289} INFO - Starting attempt 1 of 3
[2023-04-06T18:44:28.791+0400] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-06T18:44:28.815+0400] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): archive_run_dir> on 2023-04-06 14:44:02.060905+00:00
[2023-04-06T18:44:28.825+0400] {standard_task_runner.py:55} INFO - Started process 127 to run task
[2023-04-06T18:44:28.838+0400] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', '10X_sequence', 'archive_run_dir', 'testrun_06_04_2023 18_44_01', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/10X_sequence_dag.py', '--cfg-path', '/tmp/tmph6tst5el']
[2023-04-06T18:44:28.840+0400] {standard_task_runner.py:83} INFO - Job 12: Subtask archive_run_dir
[2023-04-06T18:44:28.961+0400] {task_command.py:389} INFO - Running <TaskInstance: 10X_sequence.archive_run_dir testrun_06_04_2023 18_44_01 [running]> on host eda2dce0858d
[2023-04-06T18:44:29.054+0400] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=10X_sequence
AIRFLOW_CTX_TASK_ID=archive_run_dir
AIRFLOW_CTX_EXECUTION_DATE=2023-04-06T14:44:02.060905+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=testrun_06_04_2023 18_44_01
[2023-04-06T18:44:29.095+0400] {transport.py:1873} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
[2023-04-06T18:44:29.387+0400] {transport.py:1873} INFO - Auth banner: b'~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\n           WARNING:  UNAUTHORIZED PERSONS ........ DO NOT PROCEED\n           ~~~~~~~   ~~~~~~~~~~~~~~~~~~~~          ~~~~~~~~~~~~~~\n This computer system is operated by New York University (NYU) and may be\n accessed only by authorized users.  Authorized users are granted specific,\n limited privileges in their use of the system.  The data and programs\n in this system may not be accessed, copied, modified, or disclosed without\n prior approval of NYU.  Access and use, or causing access and use, of this\n computer system by anyone other than as permitted by NYU are strictly pro-\n hibited by NYU and by law and may subject an unauthorized user, including\n unauthorized employees, to criminal and civil penalties as well as NYU-\n initiated disciplinary proceedings.  The use of this system is routinely\n monitored and recorded, and anyone accessing this system consents to such\n monitoring and recording.  Questions regarding this access policy should be\n directed (by e-mail) to AskITS@nyu.edu or (by phone) to 212-998-3333.\n Questions on other topics should be directed to COMMENT (by email) or to\n 212-998-3333 by phone.\n~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\n\n'
[2023-04-06T18:44:29.388+0400] {transport.py:1873} INFO - Authentication (publickey) successful!
[2023-04-06T18:44:29.388+0400] {ssh_helpers.py:43} INFO - Executing command: rsync -av  /scratch/gencore/testdir/jayaback /archive/gencore/testdir/
[2023-04-06T18:44:29.667+0400] {ssh_helpers.py:75} INFO - sending incremental file list
[2023-04-06T18:44:29.670+0400] {ssh_helpers.py:75} INFO - jayaback/
[2023-04-06T18:44:29.671+0400] {ssh_helpers.py:75} INFO - jayaback/10xsimple.csv
[2023-04-06T18:44:29.671+0400] {ssh_helpers.py:75} INFO - jayaback/demultiplex.sh
[2023-04-06T18:44:29.715+0400] {ssh_helpers.py:75} INFO - sent 55,858 bytes  received 61 bytes  111,838.00 bytes/sec
total size is 57,734  speedup is 1.03
[2023-04-06T18:44:29.718+0400] {ssh_helpers.py:94} INFO - Command exited with exitcode 0
[2023-04-06T18:44:29.718+0400] {python.py:177} INFO - Done. Returned value was: None
[2023-04-06T18:44:29.731+0400] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=10X_sequence, task_id=archive_run_dir, execution_date=20230406T144402, start_date=20230406T144428, end_date=20230406T144429
[2023-04-06T18:44:29.762+0400] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-06T18:44:29.782+0400] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
